code/cutscene/ogg/OggDecoder.cpp

```diff
@@ -233,7 +234,7 @@ namespace cutscene
                 vorbis_block_init(&movie.vstate, &movie.vblock);
             }
 
-            auto fps = static_cast<float>(movie.tinfo.fps_numerator) / movie.tinfo.fps_denominator;
+            fps = static_cast<float>(movie.tinfo.fps_numerator) / movie.tinfo.fps_denominator;
```

Comment:
Why did you move the declaration to the beginning of the function? In-place declarations are a lot clearer IMO.